docker/exporter/export_runner.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""OSV Exporter."""
+import argparse
+import concurrent.futures
+import logging
+import os
+import subprocess
+
+from google.cloud import ndb
+
+import osv
+import osv.logs
+
+DEFAULT_WORK_DIR = '/work'
+DEFAULT_EXPORT_BUCKET = 'osv-vulnerabilities'
+DEFAULT_EXPORT_PROCESSES = 7
+
+
+def main():
+  parser = argparse.ArgumentParser(description='Exporter')
+  parser.add_argument(
+      '--work_dir', help='Working directory', default=DEFAULT_WORK_DIR)
+  parser.add_argument(
+      '--bucket',
+      help='Bucket name to export to',
+      default=DEFAULT_EXPORT_BUCKET)
+  parser.add_argument(
+      '--processes',
+      help='Maximum number of parallel exports',
+      default=DEFAULT_EXPORT_PROCESSES)
```

Comment:
It might be tidier to default to `os.cpu_count()` here, that way this just does the right/intended thing if the number of CPUs is increased in the Kubernetes job spec?

Reply:
Good point, updated to use os.cpu_count() by default, and falling back to DEFAULT_EXPORT_PROCESSES if failing to get CPU count.